app/views/char_types/_form.html.erb

```diff
@@ -1,4 +1,5 @@
-<%= form_for :char_type, url: char_types_path, :html => { :class => "form-horizontal", :multipart => true } do |f| %>
+<%= form_for :char_type, :url => char_types_path, 
```

Comment:
I'm not sure you should be reusing this form, I might be wrong, lets talk about this too.